diesel/src/mysql_like/backend.rs

```diff
@@ -0,0 +1,25 @@
+//! The MySQL backend
+use byteorder::NativeEndian;
+
+use mysql::bind_collector::MysqlBindCollector;
+use mysql::query_builder::MysqlQueryBuilder;
+use mysql::MysqlType;
+use backend::*;
+use sql_types::TypeMetadata;
+
+pub trait MysqlLikeBackend: Backend<RawValue=[u8]> {}
+
+impl<DB: MysqlLikeBackend> Backend for DB {
+    type QueryBuilder = MysqlQueryBuilder;
+    type BindCollector = MysqlBindCollector;
+    type RawValue = [u8];
+    type ByteOrder = NativeEndian;
+}
```

Comment:
Rust is so cool. I'm surprised this self referential trait relationship even compiled